src/PrestaShopBundle/Resources/config/services/adapter/product.yml

```diff
@@ -230,3 +249,25 @@ services:
         tags:
             - name: tactician.handler
               command: PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductSeoCommand
+
```

Comment:
I wonder if we should separate handlers services from other services in two yml files?

Reply:
Maybe. But its already separated by domain (product) and it's in adapter ns, not sure if its worth it. :man_shrugging:

Reply:
Yeah probably, it was just a suggestion to avoid having too big yml files, but the important thing is that we can find the services definition easily anyway